opentuner/search/technique.py

```diff
@@ -204,15 +223,20 @@ def call_main_generator(self):
                                     self.pending_tests)
         if self.pending_tests:
           self.rounds_since_novel_request = 0
-          yield None # wait
+          yield False # wait
 
 #list of all techniques
 the_registry = list()
 
+#list of composable techniques
+the_composable_registry = list()
 
 def register(t):
   the_registry.append(t)
 
+def register_composable(t):
```

Comment:
Suppose I came along and added a new class of model-driven techniques.  We should try to make this new code extensible such that I don't need to create a `register_model_driven`, `register_hill_climber`, etc.  Can we come up with a more abstract concept specific to composable.